data/content.go

```diff
@@ -494,13 +510,7 @@ var (
 	fs_Filterscript_Matcher_0              = substring.Regexp(`(?m)#include|#pragma\s+(rs|version)|__attribute__`)
 	gs_Gosu_Matcher_0                      = substring.Regexp(`(?m)^uses java\.`)
 	h_ObjectiveDashC_Matcher_0             = substring.Regexp(`(?m)^\s*(@(interface|class|protocol|property|end|synchronised|selector|implementation)\b|#import\s+.+\.h[">])`)
-	h_CPlusPlus_Matcher_0                  = substring.Regexp(`(?m)^\s*#\s*include <(cstdint|string|vector|map|list|array|bitset|queue|stack|forward_list|unordered_map|unordered_set|(i|o|io)stream)>`)
-	h_CPlusPlus_Matcher_1                  = substring.Regexp(`(?m)^\s*template\s*<`)
-	h_CPlusPlus_Matcher_2                  = substring.Regexp(`(?m)^[ \t]*try`)
-	h_CPlusPlus_Matcher_3                  = substring.Regexp(`(?m)^[ \t]*catch\s*\(`)
-	h_CPlusPlus_Matcher_4                  = substring.Regexp(`(?m)^[ \t]*(class|(using[ \t]+)?namespace)\s+\w+`)
-	h_CPlusPlus_Matcher_5                  = substring.Regexp(`(?m)^[ \t]*(private|public|protected):$`)
-	h_CPlusPlus_Matcher_6                  = substring.Regexp(`(?m)std::\w+`)
+	h_CPlusPlus_Matcher_0                  = substring.Regexp(`(?m)CPlusPlusRegex`)
```

Comment:
This certainly doesn't work

Reply:
Nice catch. Something needs updating in the parser.